packages/cozy-clisk/src/contentscript/ContentScript.spec.js

```diff
@@ -5,11 +5,11 @@ describe('ContentScript', () => {
   describe('saveFiles', () => {
     it('should call launcher saveFiles with given nominal options', async () => {
       const contentScript = new ContentScript()
-      contentScript.setContentScriptType(PILOT_TYPE)
       contentScript.bridge = {
         call: jest.fn(),
         emit: jest.fn()
       }
+      contentScript.contentScriptType = PILOT_TYPE
```

Comment:
why this change?

Reply:
To avoid the call to setContentScript in this test, since setContentScript does more now.

Reply:
What else does it do?